app/scripts/HorizontalMultivecTrack.js

```diff
@@ -141,16 +141,6 @@ export default class HorizontalMultivecTrack extends HeatmapTiledPixiTrack {
     return `${tile.join('.')}.${tile.mirrored}`;
   }
 
```

Comment:
It is used, but it is being defined twice: Unfold the code just above. My understanding is that when there are multiple definitions in a class, the last one overrides the earlier ones... but in this case, the first definition was touched by @flekschas more recently than the second. I don't know that we actually want that change in behavior, but I wanted to surface the issue.

Reply:
The second one is actually correct. The first instance is from a heatmap track. Hi-C matrices are symmetrical so we only store one triangle. Mirrored tiles are on the other triangle. We actually don't ever request mirrored tiles from the server because that would be a waste of bandwidth. We request one triangle and then copy and rotate the texture.

So you can delete the previous function and leave the one that's currently deleted (the one without the mirror and transform properties) present.

Reply:
done.